server/tests/entity.rs

```diff
@@ -371,3 +371,100 @@ mod create_mutation {
         .await;
     }
 }
+
+#[cfg(test)]
+mod deleted_entities_query {
+    use test_utils::*;
+
+    #[actix_rt::test]
+    async fn gives_back_first_deleted_entities() {
+        let mut transaction = begin_transaction().await;
+        let first: i32 = 3;
+        let response = Message::new("DeletedEntitiesQuery", json!({ "first": first }))
+            .execute_on(&mut transaction)
+            .await;
+
+        assert_ok_with(response, |result| {
+            assert_has_length(&result["deletedEntities"], first as usize);
+            assert_json_include!(
+                actual: &result["deletedEntities"][0],
+                expected: json!({
+                    "id": 14809,
+                    "dateOfDeletion": "2014-03-10T13:26:52+01:00",
+                })
```

Comment:
Just a minor thing: `assert_json_include!` is to be used when we don't actually have to know what is going to come from the result (v.g. when creating an entity, we become automatically the field `date` filled in, which we cannot know beforehand what value it is going to get). For this use case though an `assert_json` would be enough like in https://github.com/serlo/serlo.org-database-layer/blob/main/server/tests/events.rs#L14-L24

It is not a bad thing to leave it as it is though.